compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -2541,7 +2549,7 @@ private Result seekMatchInExpressionRhs(int lookahead, int currentDepth, int cur
         } else if (parentCtx == ParserRuleContext.ON_CLAUSE) {
             nextContext = ParserRuleContext.EQUALS_KEYWORD;
         } else {
-            throw new IllegalStateException(parentCtx.toString());
+            throw new IllegalStateException("seekMatchInExpressionRhs found: " + parentCtx);
```

Comment:
btw these changes are done in order to quickly spot the location of the bug when tests are falling.
e.g.
![Screenshot from 2021-03-16 19-21-14](https://user-images.githubusercontent.com/39232462/112441166-24041400-8d71-11eb-8b8e-dcec690a3990.png)